src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -145,7 +145,7 @@
 
     <!-- Default behavior is to disable OuterLoop and failing tests if not specified in WithCategories. -->
     <ItemGroup>
-      <DefaultNoCategories Include="OuterLoop" />
+      <DefaultNoCategories Condition="'$(Outerloop)'!='true'" Include="OuterLoop" />
```

Comment:
I don't think so. That would be significantly more verbose than "/p:Outerloop=true". 

I'd prefer we keep it as simple as possible for Outerloop since it's such a common trait to want to run.